### DISCUSSION TITLE
License of the Image of an icon itself

### DISCUSSION THREAD

**Discussion by sOER-Frank** [lucide-icons/lucide#1021] [Q&A]
On the[ Lucide page](https://lucide.dev/license) it says under "Licence":

> Lucide License
> ISC License
> Copyright (c) for portions of Lucide are held by Cole Bemis 2013-2022 as part of Feather (MIT). All other copyright (c) for Lucide are held by Lucide Contributors 2022.
> 
> Permission to use, copy, modify, and/or distribute this software for any purpose with or without fee is hereby granted, provided that the above copyright notice and this permission notice appear in all copies.
> 
> THE SOFTWARE IS PROVIDED "AS IS" AND THE AUTHOR DISCLAIMS ALL WARRANTIES WITH REGARD TO THIS SOFTWARE INCLUDING ALL IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS. IN NO EVENT SHALL THE AUTHOR BE LIABLE FOR ANY SPECIAL, DIRECT, INDIRECT, OR CONSEQUENTIAL DAMAGES OR ANY DAMAGES WHATSOEVER RESULTING FROM LOSS OF USE, DATA OR PROFITS, WHETHER IN AN ACTION OF CONTRACT, NEGLIGENCE OR OTHER TORTIOUS ACTION, ARISING OUT OF OR IN CONNECTION WITH THE USE OR PERFORMANCE OF THIS SOFTWARE.

But as I read it, the licence refers to the software and not to the image of an icon. I would like to use icons in Open Educational Resources and grant a Creative Commons licence for these materials. Is this allowed? If so, which licence?

**Comment by bring-shrubbery** [comment]
**I'm NOT official Lucide representative, NOR am I a layer or something, BUT** as far as I know the "image representation" is protected under Copyright, but the License allows you to use and modify this image in any way you want, you just need to attribute creators and not say that you created the image (which would be copyright infringement).

Also, everything you see on the screen is "software". Don't quote me on that though. I think if you're unsure about legality of any usage the best way to go is to talk with a qualified lawyer. I think people here are as confused about this stuff as you and I 😃
